force-app/tdtm/classes/TDTM_Runnable.cls

```diff
@@ -195,6 +196,81 @@ global abstract class TDTM_Runnable {
         }
     }
     
+    /**
```

Comment:
I like the way you've handled this.  It ought to make it easier for us to generalize this solution if we need to.